release/air_tests/air_benchmarks/mlperf-train/compute_cpu_16.yaml

```diff
@@ -6,15 +6,17 @@ max_workers: 0
 head_node_type:
     name: head_node
     instance_type: m5.4xlarge
-    BlockDeviceMappings:
-        - DeviceName: /dev/sda1
-          Ebs:
-              VolumeSize: 300
-              DeleteOnTermination: true
 
 worker_node_types:
     - name: worker_node
       instance_type: m5.xlarge
       min_workers: 0
       max_workers: 0
       use_spot: false
+
+aws:
+    BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+            DeleteOnTermination: true
+            VolumeSize: 400
```

Comment:
so 300 is not enough?

Reply:
Peak usage was 326 GB - this test was completely broken on master and not running at all